e2e_test/run_all_tests.sh

```diff
@@ -2,19 +2,20 @@
 set -eo pipefail
 
 SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+source "$SCRIPT_DIR/shared.sh"
+source "$SCRIPT_DIR/debug-fee-currency/lib.sh"
```

Comment:
Why do we need `lib.sh` in the test runner?

Reply:
This is a leftover from previously calling a lib function in the test runner, will remove it